src/coreComponents/mesh/MeshManager.cpp

```diff
@@ -106,7 +106,13 @@ void MeshManager::importFields( DomainPartition & domain )
   forSubGroups< MeshGeneratorBase >( [&domain]( MeshGeneratorBase const & generator )
   {
     if( !domain.hasMeshBody( generator.getName() ) )
+    {
       return;
+    }
+    else if( domain.getMeshBody( generator.getName() ).hasParticles() )
```

Comment:
I am not sure I understand this. how does having particle relate to fields import?

Reply:
The addition of VTK field importation broke my branch and I was not able to figure out why, so I put this is in to circumvent it. We will want to use VTK input for particles someday, but that's beyond the scope of this PR.